guides/GUIDE_FOR_LIBRARY_AUTHORS.md

```diff
@@ -143,6 +149,32 @@ Sets the current screen's available orientations and forces rotation if current
 - `landscape_right`
 
 Defaults to `default` on iOS.
+### `sheetAllowedDetents` (iOS only)
+
+Describes heights where a sheet can rest. 
```

Comment:
Maybe we should you add the available options similar to e.g. `stackAnimation`

Reply:
Same for the other props

Reply:
[bcfbc09](https://github.com/software-mansion/react-native-screens/pull/1649/commits/bcfbc093d1237092400b6e715c3967e5d11d6e08)